apps/ai-content-generator/src/configs/prompts/contentPrompt.ts

```diff
@@ -0,0 +1,3 @@
+const contentPrompt = (prompt: string) => `Write a blog post in 400 words about "${prompt}"`;
```

Comment:
Why can't all these prompts live in one file and be exported accordingly?

Reply:
I'm cool with condensing these into one file since they're small!